doc/releases/release-notes-2.3.rst

```diff
@@ -200,7 +206,10 @@ Architectures
 
 * x86:
 
-  * <TBD>
+  * Instrument code for timing information
+  * Add ability for SoC to add MMU regions
```

Comment:
#25949 changed these kind of verbs to past tense, should we follow suit here

Reply:
well, this can always be done in the last pass...

Reply:
Sure, I just noticed it and commented.